WordpressSync/endpoints.schema.json

```diff
@@ -91,6 +115,10 @@
                       "wordpress/pages"
                     ]
                   },
+                  "RelativePaths": {
```

Comment:
Recommend we add a verb to the name like `RewriteRelativePaths`.  My initial reading of this I was thinking it was describing the path data rather that authorizing a new form of automation.